selenium-testing/pom.xml

```diff
@@ -0,0 +1,70 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>com.testng.selenium</groupId>
```

Comment:
See other comments on code packages.

Reply:
Changed in pom.xml: https://github.com/bcgov/ecc-ofm/pull/319/commits/5247c4beed17cdb26a54db8c56bc613625051a32